content/en/Features/multi-region/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Multi-Region"
+linkTitle: "Multi-Region"
+weight: 5
+description: How to use Opta to deploy multi-region environments
+---
+
+## Warning: Here be Dragons!
```

Comment:
this section is good, may be we can redirect them to multi-zone ([here](https://docs.opta.dev/features/networking/high_availability/
)) which is much easier, and would address their requirement for increased availability.